### ISSUE TITLE
Reference Error: number is not defined

### ISSUE THREAD

**Issue by phil-kt** [cedricdelpoux/react-svg-line-chart#9]
My code:

```
render() {
    const data = [
      {x: 1, y: 144.11}, {x: 2, y: 145.06}, {x: 3, y: 144.73}, {x: 4, y: 145.53}, {x: 5, y: 145.87}
    ];



    return(
     <LineChart data={data}/>
    )
}
```

**Comment by cedricdelpoux** [comment]
Thank you for reporting this. I identified the problem. I will fix it soon

**Comment by cedricdelpoux** [comment]
@LinkCable I just fixed it in `v1.0.6`.

Thank you
